Lib/email/_header_value_parser.py

```diff
@@ -1360,6 +1360,9 @@ def get_word(value):
         leader, value = get_cfws(value)
     else:
         leader = None
+    if not value:
+        raise errors.HeaderParseError("Expected 'atom' or 'quoted-string' "
```

Comment:
I'd probably put a newline break after the open paren to see if you can avoid the implicit concatenation.

Reply:
Done.